src/platform/forms-system/src/js/components/SchemaForm.jsx

```diff
@@ -165,7 +165,7 @@ class SchemaForm extends React.Component {
   }
 
   validate(formData, errors) {
-    const { schema, uiSchema, appStateData } = this.props;
+    const { schema, uiSchema, appStateData, getFormData } = this.props;
```

Comment:
I'm not sure why this is needed in validation functions. The combo of `appStateSelector` and `appStateData` allows a dev to pick and choose what (full) data to pass into validation functions. I could see included the full form data outside of validation functions.

Reply:
This is a great point. I didn't realize you could inject any form data via `appStateSelector` and `appStateData`. 
Confirmed that that DOES work to get any data into ui:validations, but is just more of a manual process/workaround for the user. Hmm.

pros:
I think the workflow with this change is easier for the consumer, adding a single ui:option.

cons: 
we already have a way to do it, but requires the user knowing about `appStateSelector` and `appStateData` and setting that up in both the config/form, and using the 7th param inside ui:validations.

I'm leaning toward going ahead with this approach, but I'll sync with you offline.

Reply:
I didn't realize it either. I would like to see an example of accessing the form data with `appStateSelector` and `appStateData` just for future reference.

Reply:
more info on `appStateSelector`:

apparently `appStateSelector` is a page property: https://github.com/department-of-veterans-affairs/vets-website/blob/672a26ae1a6197df18f81cf409123c2d1f429b0e/src/applications/appeals/10182/config/form.js#L170
which you can choose any extra data you want https://github.com/department-of-veterans-affairs/vets-website/blob/672a26ae1a6197df18f81cf409123c2d1f429b0e/src/applications/appeals/shared/utils/issues.jsx#L180
and then use it in `ui:validations` function https://github.com/department-of-veterans-affairs/vets-website/blob/672a26ae1a6197df18f81cf409123c2d1f429b0e/src/applications/appeals/10182/validations/issues.js#L20

It's also passed to CustomPage, but not any fields.